plugins/pyenv-virtualenvwrapper/pyenv-virtualenvwrapper.plugin.zsh

```diff
@@ -0,0 +1,51 @@
+eval "$(pyenv init -)"
```

Comment:
Pyenv initialization is handled in the pyenv plugin. I'd recommend loading it directly from here. I think there's an oh-my-zsh root directory variable that you can reference for portability.